File: scalechain-net/src/test/scala/io/scalechain/blockchain/net/handler/HeadersMessageHandlerSpec.scala
package io.scalechain.blockchain.net.handler

import java.io.File

import io.netty.channel.embedded.EmbeddedChannel
import org.scalatest._


class HeadersMessageHandlerSpec extends MessageHandlerTestTrait with ShouldMatchers {
  this: Suite =>

  val testPath = new File("./target/unittests-HeadersMessageHandlerSpec/")

  "handle" should "" in {
  }
}

File: scalechain-net/src/test/kotlin/io/scalechain/blockchain/net/handler/HeadersMessageHandlerSpec.kt
package io.scalechain.blockchain.net.handler

import io.kotlintest.KTestJUnitRunner
import io.kotlintest.matchers.Matchers
import org.junit.runner.RunWith
import java.io.File

@RunWith(KTestJUnitRunner::class)
class HeadersMessageHandlerSpec : MessageHandlerTestTrait(), Matchers {

  override val testPath = File("./build/unittests-HeadersMessageHandlerSpec/")

  init {
    "handle" should "" {
    }
  }
}